ui/de-lib/src/main/java/org/iplantc/de/theme/base/client/desktop/views/taskButton/TaskButtonCell.css

```diff
@@ -5,6 +5,26 @@
  *
  * http://www.sencha.com/products/gxt/license/
  */
+@font-face {
+  font-family: TextaRegular;
+  src: url(../../Texta_Font/Texta-Regular.otf);
+}
+
+@font-face {
+  font-family: TextaBlack;
+  src: url(../../Texta_Font/Texta-Black.otf);
+}
+
+@font-face {
+  font-family: TextaBold;
+  src: url(../../Texta_Font/Texta-Bold.otf);
+}
+
+@font-face {
+  font-family: TextaItalic;
+  src: url(../../Texta_Font/Texta-BlackItalic.otf);
+}
```

Comment:
Yeah, I think some of these are left over from when we were experimenting with different variations of the font, and then I copy-pasted everywhere else.  I can clean these up.